src/apps/common/submodules/numbers/numbers.js

```diff
@@ -1539,6 +1612,21 @@ define(function(require) {
 			});
 		},
 
+		listAccountExternalNumbers: function(accountId, success) {
```

Comment:
All the functions in a submodule should have the submodule name as prefix. In this case it would be something like `numbersListExternalNumbers`. Also, having the API call for `externalNumbers.list` as a separate function, it would be a good time to use it in here: https://github.com/2600hz/monster-ui/blob/master/src/apps/common/submodules/numbers/numbers.js#L1729